days/day003/src/main/java/com/thegreatapi/ahundreddaysofjava/day003/Day003.java

```diff
@@ -1,28 +1,48 @@
 package com.thegreatapi.ahundreddaysofjava.day003;
 
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
 import java.time.LocalTime;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-public class Day003 {
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-    private final ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+class Day003Test {
```

Comment:
When converting to a test class, it should be moved to the test directory. Please use https://github.com/hbelmiro/100DaysOfJava/pull/62 as a reference.

Reply:
> So you want me to delete the main folder and create a new test folder, and then make the necessary changes in it?

Yeah! Also, you need to update the link in `README.adoc` to the new file.